tests/installation/prepare_maintenance_env.pm

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Prepare test repositories for children jobs.
+# - Disable Nvidia repos that may have been automatically added as part of autoinstall.
+# - Import maintenance repos gpg keys
+# - Run zypper patch
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use utils;
+use testapi;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    quit_packagekit unless check_var('DESKTOP', 'textmode');
+    # Disable gpg check for all maintenance repositories.
+    zypper_call('--gpg-auto-import-keys ref');
```

Comment:
we should keep the extra params and the comment:
```
    # refresh repositories, inf 106 is accepted because repositories with test
    # can be removed before test start
    zypper_call('ref', timeout => 1400, exitcode => [0, 106]);
```